src/bereikbaarheid/signals.py

```diff
@@ -0,0 +1,26 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from bereikbaarheid.resources.utils import refresh_materialized
+
+from .models import Lastbeperking, VerkeersBord, Verrijking
+
+
+@receiver(post_save, sender=Lastbeperking)
+@receiver(post_save, sender=VerkeersBord)
+@receiver(post_save, sender=Verrijking)
+def import_post_save(sender, instance, **kwargs):
+    """When add/change by admin-panel: Refresh materialized views"""
```

Comment:
ik zie dit nergens worden aangeroepen in deze commit. hoe wordt deze functie aangeroepen?

Reply:
via apps.py